internal/service/credentials/resource_credential_issuance_rule.go

```diff
@@ -0,0 +1,842 @@
+	EnvironmentId              types.String `tfsdk:"environment_id"`
+	CredentialTypeId           types.String `tfsdk:"credential_type_id"`
+	DigitalWalletApplicationId types.String `tfsdk:"digital_wallet_application_id"`
+	Automation                 types.Object `tfsdk:"automation"`
+	Filter                     types.Object `tfsdk:"filter"`
+	Notification               types.Object `tfsdk:"notification"`
+	Status                     types.String `tfsdk:"status"`
+}
+
+type FilterModel struct {
+	GroupIds      types.Set    `tfsdk:"group_ids"`
+	PopulationIds types.Set    `tfsdk:"population_ids"`
+	Scim          types.String `tfsdk:"scim"`
+}
+
+type AutomationModel struct {
+	Issue  types.String `tfsdk:"issue"`
+	Revoke types.String `tfsdk:"revoke"`
+	Update types.String `tfsdk:"update"`
+}
+
+type NotificationModel struct {
+	Methods  types.Set    `tfsdk:"methods"`
+	Template types.Object `tfsdk:"template"`
+}
+
+type NotificationTemplateModel struct {
+	Locale  types.String `tfsdk:"locale"`
+	Variant types.String `tfsdk:"variant"`
+}
+
+var (
+	filterServiceTFObjectTypes = map[string]attr.Type{
+		"group_ids":      types.SetType{ElemType: types.StringType},
+		"population_ids": types.SetType{ElemType: types.StringType},
+		"scim":           types.StringType,
+	}
+
+	automationServiceTFObjectTypes = map[string]attr.Type{
+		"issue":  types.StringType,
+		"revoke": types.StringType,
+		"update": types.StringType,
+	}
+
+	notificationServiceTFObjectTypes = map[string]attr.Type{
+		"methods":  types.SetType{ElemType: types.StringType},
+		"template": types.ObjectType{AttrTypes: notificationTemplateServiceTFObjectTypes},
+	}
+
+	notificationTemplateServiceTFObjectTypes = map[string]attr.Type{
+		"locale":  types.StringType,
+		"variant": types.StringType,
+	}
+)
+
+// Framework interfaces
+var (
+	_ resource.Resource                = &CredentialIssuanceRuleResource{}
+	_ resource.ResourceWithConfigure   = &CredentialIssuanceRuleResource{}
+	_ resource.ResourceWithImportState = &CredentialIssuanceRuleResource{}
+)
+
+// New Object
+func NewCredentialIssuanceRuleResource() resource.Resource {
+	return &CredentialIssuanceRuleResource{}
+}
+
+// Metadata
+func (r *CredentialIssuanceRuleResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_credential_issuance_rule"
+}
+
+func (r *CredentialIssuanceRuleResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+
+	statusdDescriptionFmt := "Status of the credential issuance rule. Can be `ACTIVE` or `DISABLED`."
+	statusDescription := framework.SchemaDescription{
+		MarkdownDescription: statusdDescriptionFmt,
+		Description:         strings.ReplaceAll(statusdDescriptionFmt, "`", "\""),
+	}
+
+	filterDescriptionFmt := "Contains one and only one filter (`.groupIds`, `.populationIds`, or `.scim`) that selects the users to which the credential issuance rule applies."
```

Comment:
"`.groupIds`, `.populationIds`" should probably be the TF schema equivalent

Reply:
resolved